config/routes.rb

```diff
@@ -6,9 +6,13 @@
   resources :directors, shallow: true do
     resources :movies
   end
+
   resources :movies, only: [:index]
   resources :actors
 
+  resources :genres, only: [:index, :show]
+  post '/genres', to: 'admin/genres#create'
```

Comment:
This works, you could also have created an admin namespace and put resources :genres, only: [:create] in it.